resources/troubleshooting-guides/push_notification_troubleshooting.md

```diff
@@ -0,0 +1,172 @@
+   
+   6. If you are using Charles, verify that the push token has successfully synced with the ECID service.  
+   7. Check for the **demdex request**, which is marked with the red line in the screenshot below.  
+   8. Verify the successful response(200) for this network call.       
+ 
+ ![ECID network request for push token sync](../../.gitbook/assets/charles_demdex_call_ios.png "ECID network request for push token sync.")
+ 
+  {% endtab %}
+  {% endtabs %}
+
+
+ ## Troubleshooting push notification events tracking  
+
+The following events are related to the tracked push notifications:
+  * Impression (Notification is delivered.)
+  * Click (User clicked the notification.)
+  * Open (App opened in response to user's click on notification.)  
+  For more information about implementing the tracking of push notification events, see [Push Tracking](https://helpx.adobe.com/campaign/kb/push-tracking.html).  
+  
+###  Troubleshooting using Project Griffon  
+
+  You can verify the push token sync with the ECID service in Project Griffon.  
+  
+  1. Connect your app to a Griffon session.  
+  2. Send a push notification to app from your Campaign instance.  
+  To know more about how to send push notification, see [Preparing and sending a push notification](https://docs.adobe.com/content/help/en/campaign-standard/using/communication-channels/push-notifications/preparing-and-sending-a-push-notification.html).
+  3. Receive a push notification in the device.  
+  4. Click on the push notification to launch the app.  
+  5. In the list of events, verify that you have an event with type CollectData.  
+  6. In the Details panel on the right, verify the value of the action.  
+  The value of the action should be 7 for impression, 2 for the click, and 1 for open.  
+  It is shown in the screenshots below.  
+  
+   ![Shows Push notification impression tracking](../../.gitbook/assets/push_tracking_impression.png "Shows Push notification impression tracking.")
+   
+   Shows tracking of the _impression_ event. It is indicated by action 7.
+   
+   ![Shows Push notification click tracking](../../.gitbook/assets/push_tracking_click.png "Shows Push notification click tracking.")
+   
+   Shows tracking of the _click_ event. It is indicated by action 2.
+   
+   ![Shows Push notification open tracking](../../.gitbook/assets/push_tracking_open.png "Shows Push notification open tracking.")
+   
+   Shows tracking of the _open_ event. It is indicated by action 1.
+  
+  
+###  Troubleshooting using Charles  
+
+To verify that a successful network call is made to track the push notification interaction event, look for the GET request with the query string `id=broadlogID,deliveryID,action` to the Campaign server, see screenshots below for more details about what to look.  
+
+Screenshot for the impression event tracking(action value 7)
```

Comment:
Please reword this screenshot text to:
"Impression event tracking (action value 7)"